test/distributed/fsdp/test_fsdp_exec_order.py

```diff
@@ -88,6 +88,11 @@ def wrap(sharding_strategy: ShardingStrategy, device: torch.device):
 
 
 class TestFSDPExecOrder(FSDPTest):
+    
+    def setUp(self):
+        super().setUp()
+        dist.set_debug_level(dist.DebugLevel.INFO)
```

Comment:
Do you prefer we run execution order validation on all unit tests or only on the ones in `test_fsdp_exec_order.py`? If you prefer we run it for all, I will adjust my PR as well.

Reply:
Hmm, maybe let's do it here and just test_fsdp_core, do you think that will be representative enough?

Reply:
That sounds good to me!